test/unit/rules/require-input-label-test.js

```diff
@@ -91,12 +94,12 @@ generateRuleTests({
       },
     },
     {
-      template: '<label><input></label>',
+      template: '<label><input /></label>',
```

Comment:
Please reset this to HTML syntax (not XHTML). I think having an XHTML example would be perfectly fine, but it should be a new test (not mutating an existing / unrelated one).

Reply:
Ok, changed it back.